tests/factories.py

```diff
@@ -11,27 +11,55 @@
 from django.utils.timezone import utc
 
 from django.contrib.auth import get_user_model
-#from django_countries.fields import CountryField
+from django.contrib.sites.models import Site
+
 import factory
 from factory import fuzzy
 from factory.django import DjangoModelFactory
 
 from openedx.core.djangoapps.content.course_overviews.models import (
     CourseOverview,
 )
-
+# from openedx.core.djangoapps.site_configuration.models import SiteConfiguration
 from openedx.core.djangoapps.xmodule_django.models import CourseKeyField
 from certificates.models import GeneratedCertificate
 from courseware.models import StudentModule
 from student.models import CourseAccessRole, CourseEnrollment, UserProfile
 from lms.djangoapps.teams.models import CourseTeam, CourseTeamMembership
 
+import organizations
+# from organizations import tests as org_tests
 from figures.helpers import as_course_key
 from figures.models import CourseDailyMetrics, SiteDailyMetrics
 
+from tests.helpers import organizations_support_sites
+
+
 COURSE_ID_STR_TEMPLATE = 'course-v1:StarFleetAcademy+SFA{}+2161'
 
 
+
+# class SiteConfigurationFactory(factory.DjangoModelFactory):
```

Comment:
@johnbaldwin Please remove if we don't need it.

Reply:
will do